docs/configuration/tempo-config.md

```diff
@@ -113,7 +113,7 @@ automatic_logging:
   # This feature only applies when `backend = logs_instance`
   [ labels: <string array> ]
   overrides:
-    [ logs_instance_tag: <string> | default = "tempo" ]
+    [ logs_instance_tag: <string> | default = "traces" ]
```

Comment:
IMO I think until we consistently use traces throughout the docs, everything in this file _except_ this line and the comment for `namespace` should be left alone. WDYT?